readthedocs/settings/docker_compose.py

```diff
@@ -24,6 +24,14 @@ class DockerBaseSettings(CommunityDevSettings):
     MULTIPLE_APP_SERVERS = ['web']
     MULTIPLE_BUILD_SERVERS = ['build']
 
+    # https://docs.docker.com/engine/reference/commandline/run/#add-entries-to-container-hosts-file---add-host
+    # export HOSTIP=`ip -4 addr show scope global dev wlp4s0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
+    HOSTIP = os.environ.get('HOSTIP')
```

Comment:
I'd like to use `host.docker.internal` if we're on a mac. That way there's no configuration needed.

Reply:
I pushed a change for this.